source/process/release-process.md

```diff
@@ -258,6 +259,7 @@ Once final dot release build is ready to cut:
       - Test the download links before and after updating the page 
     - Update [Mattermost pricing page](https://about.mattermost.com/pricing/) if anything has changed
     - Add the download links to http://docs.mattermost.com/administration/upgrade.html#version-archive  
+    - Update [mattermost-docker](https://github.com/mattermost/mattermost-docker) version 
```

Comment:
Propose we link directly to the file, if this is it? https://github.com/mattermost/mattermost-docker/blob/master/app/Dockerfile